contrib/container/maptool/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Dockerfile that imports the current repo checkout build it in a builder
+# container and uses cpack to get the deb to import that into the runtime container and install it
+FROM debian:stable-slim AS builder
+
+# Set working directory
+WORKDIR /app
+
+# Copy the current repo into the container
+COPY . .
+
+# Install dependencies
+RUN ./scripts/setup_common_requirements.sh && apt-get install -y libpng-dev libfreetype6-dev libdbus-glib-1-dev libgtk2.0-dev curl
+
+# Build the project
+RUN mkdir build && cd build && \
+    cmake -Dsvg2png_scaling=-1 -Dsvg2png_scaling_flag=-1 -Dsvg2png_scaling_nav=-1 -Dsvg2png_scaling_textures=-1 .. && \
+    make maptool&& \
+    cpack -G DEB
+
+# Runtime container
+FROM debian:stable-slim
```

Comment:
personally, I subscribe to reproducible builds, which would require to set a specific version here. In absence of renovate / dependabot, I think this is fine for now as is, and maybe we should consider using bots and in the future shifting over to more specific tags to help with reproducability.